app/controllers/concerns/sortable.rb

```diff
@@ -0,0 +1,29 @@
+module Sortable
+  extend ActiveSupport::Concern
+
+  def sort
+    sortable_class.transaction do
+      sort_params[:sortable_ids].each_with_index do |id, index|
+        sortable_records.where(id: id.to_i).update_all({ position: index + 1, updated_at: Time.now })
+      end
+    end
+
+    head :ok
+  end
+
+  private
+
+  def sort_params
+    params.permit(sortable_ids: [])
+  end
+
+  def sortable_class
```

Comment:
If `transaction` is the only purpose of this method then we don't need it since we can just call `ActiveRecord::Base.transaction` regardless of the model type